Apps/frontend/src/views/SineWave.svelte

```diff
@@ -65,10 +74,11 @@
   });
 
   const newFrame = () => {
-    update();
-    webGLPlot.update();
-    window.requestAnimationFrame(newFrame);
+      update();
+      webGLPlot.update();
+      window.requestAnimationFrame(newFrame);
   }
 </script>
 
 <canvas bind:this={canvasElement} />
+<StartStopButton on:startStop={async (event)=> {hasStarted = event.detail.buttonValue}} />
```

Comment:
The component should be placed in the Oscilloscope.svelte component in my opinion.